src/main/java/ca/on/oicr/gsi/dimsum/service/CaseService.java

```diff
@@ -506,14 +506,20 @@ private Stream<TestTableView> filterTestTableViews(List<Case> cases,
 
   private Map<Case, List<Test>> getFilteredCaseAndTest(List<Case> cases,
       Collection<CaseFilter> filters) {
+    if (filters == null) {
+      throw new NullPointerException("Filters cannot be null");
+    } else if (filters.isEmpty()) {
+      throw new IllegalStateException("Filters cannot be empty");
+    }
     Map<Case, List<Test>> testsByCase = new HashMap<>();
     List<Case> filteredCases = filterCases(cases, filters).toList();
     if (filters != null && !filters.isEmpty()) {
```

Comment:
Don't need this check anymore

Reply:
You don't need to check here because you were checking above. You removed the above checks instead of this one though